lib/libutee/tee_api_objects.c

```diff
@@ -812,7 +819,7 @@ TEE_Result TEE_StartPersistentObjectEnumerator(TEE_ObjectEnumHandle
 
 TEE_Result TEE_GetNextPersistentObject(TEE_ObjectEnumHandle objectEnumerator,
 				       TEE_ObjectInfo *objectInfo,
-				       void *objectID, uint32_t *objectIDLen)
+				       void *objectID, size_t *objectIDLen)
```

Comment:
No, this can't be solved by casting, not for a 64-bit target.

Reply:
> No, this can't be solved by casting, not for a 64-bit target.

Agreed, so a temporary variable is needed, isn't it? Consider a 64-bit TA built using the older devkit calling into the new libutee.so, 64 bits would get copied into a place where only 32 are expected.